src/main/java/mate/academy/service/UserService.java

```diff
@@ -14,7 +14,13 @@ public class UserService {
      * @return - user if his email is equal to passed email.
      * Return <code>null</code> if there is no suitable user
      */
-    public User findByEmail(String email) {
+    public static User findByEmail(String email) {
```

Comment:
please don't use static methods

you can call this method from AuthenticationService if you create `new UserService()` object - and call this method on that object